core/src/replay_stage.rs

```diff
@@ -3009,18 +3009,20 @@ impl ReplayStage {
                     .expect("Bank fork progress entry missing for completed bank");
 
                 let replay_stats = bank_progress.replay_stats.clone();
+                let mut are_stats_from_completed_scheduler = false;
 
                 if let Some((result, completed_execute_timings)) =
                     bank.wait_for_completed_scheduler()
                 {
+                    are_stats_from_completed_scheduler = true;
                     let metrics = ExecuteBatchesInternalMetrics::new_with_timings_from_all_threads(
                         completed_execute_timings,
                     );
                     replay_stats
                         .write()
                         .unwrap()
                         .batch_execute
-                        .accumulate(metrics);
+                        .accumulate(metrics, are_stats_from_completed_scheduler);
```

Comment:
here is just always `true` right?

Reply:
yes. I have tendency of avoiding `true` to indicate this `true` is closely related to `are_stats_from_completed_scheduler` (now called `is_unified_scheduler_enabled`... lol).